pddlgym/backend/pred_test.py

```diff
@@ -0,0 +1,46 @@
+from predicate import Predicate
```

Comment:
Add a docstring to the top of this file describing what this test is for. Also add asserts for what is expected at the end.